deploy/basic/terraform/storage.tf

```diff
@@ -116,4 +116,16 @@ resource "oci_objectstorage_preauthrequest" "mushop_lite_preauth" {
   object = "${oci_objectstorage_object.mushop_basic.object}"
 }
 
+resource "oci_objectstorage_object_lifecycle_policy" "mushop_deploy_assets_lifecycle_policy" {
+  namespace = "${data.oci_objectstorage_namespace.user_namespace.namespace}"
+  bucket    = "${oci_objectstorage_bucket.mushop.name}"
+
+  rules {
```

Comment:
No object filters ? 
 - this will delete everything in the bucket, not an issue, just making sure that's what you intended.

Reply:
The bucket is only used for the deployment. The assets does not have any use after successfully deployed.

Reply:
No action required now, but any thoughts on how this might change once the catalogue images are also added to Object Storage?
- Separate bucket?
- Or use prefixes for `images` and `deploy` assets?

Reply:
Closing PR now. Separate buckets. That's the whole idea to have buckets, to separate the sand from the water to make the castle.